Makefile

```diff
@@ -9,8 +9,8 @@ SF_WEB=hpcoder@web.sourceforge.net:/home/project-web/minsky/htdocs
 # location of TCL and TK libraries 
 TCL_PREFIX=$(shell grep TCL_PREFIX $(call search,lib*/tclConfig.sh) | cut -f2 -d\')
 TCL_VERSION=$(shell grep TCL_VERSION $(call search,lib*/tclConfig.sh) | cut -f2 -d\')
-TCL_LIB=$(dir $(shell find $(TCL_PREFIX) -name init.tcl -path "*/tcl$(TCL_VERSION)*" -print))
-TK_LIB=$(dir $(shell find $(TCL_PREFIX) -name tk.tcl -path "*/tk$(TCL_VERSION)*" -print))
+TCL_LIB=/usr/lib/tcl8.6#$(dir $(shell find $(TCL_PREFIX) -name init.tcl -path "*/tcl$(TCL_VERSION)*" -print))
+TK_LIB=/usr/lib/tk8.6#$(dir $(shell find $(TCL_PREFIX) -name tk.tcl -path "*/tk$(TCL_VERSION)*" -print))
 
```

Comment:
You should avoid adding distro specific information like this. EcoLab's configure scripts should be able to find TCL/Tk - if not, then let me know so I can fix it.

Reply:
I have removed them, they are not needed.